src/users/users.php

```diff
@@ -351,13 +350,16 @@ public function post()
 	 */
 	public function get()
 	{
-
-		$input = Factory::getApplication()->input;
+		$app = Factory::getApplication();
+		$input = $app->input;
 		$res = new stdClass();
 
-		$model = new UsersModelUsers;
-
-		$app = Factory::getApplication('administrator');
+		if (JVERSION < '4.0.0'){
+			$model = new UsersModelUsers;
```

Comment:
@vinayak-patil 
for `$model = new UsersModelUsers;` Joomla has provided a method to instantiate the model object this method allows us to maintain the default model state and it also performs some default actions which need to be done while creating the model object. The line of code comented will also work (It was already present and you haven't added it but just want to let you know hence added a comment)

// Code to load the model classes for a given extension
BaseDatabaseModel::addIncludePath(JPATH_ADMINISTRATOR . '/components/com_users/src/models/');

// Code to create object of specific model
$usersModel = BaseDatabaseModel::getInstance('Users', 'UsersModel');